google_analytics_segments

```diff
@@ -0,0 +1,5 @@
+var gaCustAud = "NOSEG";
+if(utag_data['context.user.guid'] != undefined){
+	var gaCustAud = parseInt(utag_data['context.user.guid'].split("-")[0],16) % 101;	  
```

Comment:
We'll want to use ```% 100``` for more even user segmentation, since % 101 creates 101 segments.

Is this supposed to be an extension?